terraform/modules/emr/iam.tf

```diff
@@ -281,7 +300,9 @@ data aws_iam_policy_document elastic_map_reduce_for_ec2_role {
     actions = [
       "acm-pca:GetCertficate"
```

Comment:
Do we want to retrieve the certificate directly from acm-pca, or is this an acm-managed certificate generated by acm-pca? If it's not acm-managed, then it probably should be so that we don't need to manually rotate the certificate when it expires and risk a service outage.

Reply:
This is used to grab the DKS certificates using the `acm-cert-retriever`. This is done on cluster startup (which is cycled every day), so certificate expiry shouldn't be a problem

Reply:
The certificate expiry problem isn't related to how often we retrieve the certificates. It's related to whether we request them directly from ACM-PCA (in which case we need to manually renew certificates when they expire), or whether ACM requests them from ACM-PCA on our behalf, in which case it can manage certificate renewal. This comment isn't a blocker for this PR but we need to think about the best approach as a longer-term thing.

Reply:
I think the answer to your question depends on the implementation in https://github.com/dwp/acm-pca-cert-generator retriever.

Reply:
Indeed. The tf which we needed to integrate with ACM wasn't a thing when the pca-cert-generator was written. Our very own Mr Burgess raised the pr on tf to fix that one :) I'm going to mark this thread as resolved and pick up as a more strategic thing.